README.md

```diff
@@ -158,16 +158,38 @@ pieces search query --mode fts
 ```
 
 Examines all words in a document to find matches to search criteria
+##### change the llm model you are using:
 
+Lists all the avaiable models on your machine with their indices to choose from .
+
+```bash
+pieces change_model 
```

Comment:
my suggestion would perhaps be to keep the name as `pieces list_models `
What are your thoughts @mason-at-pieces / @hal-8999-alpha

Reply:
`pieces list models` is still working u can list the models same as change model

Reply:
I believe 

```bash
  pieces list models
  ```
  
  Should be sufficient to list the models.
  
  and then we can use pieces change_model [MODEL_INDEX] to select the model.
  
  What we can do here is remove this command and add a bit more description to the change model command to tell the user to in order to change the model they should first list the models via the list model command

Reply:
I'd agree with Mark. 

list models then select from the menu would keep it uniform.

Reply:
I'd agree with Mark.

list models then select from the menu would keep it uniform.